CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 0.5.0 - 2020-12-01
+
+### Added
+
+- Publish local projects directly to GitHub
+- Advanced creation to have more control over creation
```

Comment:
What about something like this?
```suggestion
- Advanced creation provides more control when creating static web apps
```

Reply:
I always read the lines with added prefixed in front of it i.e. Added advanced creation provides more control... 

That being said, I think "to provide more control" sounds good to me